tools/experiments/experiments.html

```diff
@@ -112,6 +127,21 @@
       display: block;
     }
 
+    input#rtv {
+      border: solid 1px gray;
+      width: calc(100% - 150px);
+      padding: 10px;
+    }
+
+    button#rtv-submit {
+      background: #aaa;
+      margin: 5px;
+    }
+
+    button#rtv-submit[disabled] {
```

Comment:
Nit: drop the element selectors, ID is far more selective anyway (and it would still work if buttons were replaced with anchors or something else)

Reply:
Done